docs/tech-reference/iso-rsync-distributor.rst

```diff
@@ -57,8 +57,8 @@ The ``distributor_config`` contains a ``remote`` section with the following sett
 
 ``root``
   The absolute path to the remote root directory where all the data (content and published content)
-  lives. This is the remote equivalent to ``/var/lib/pulp``. The repo id is appended to the
-  ``root`` path to determine the location of published repository.
+  lives. This is the remote equivalent to ``/var/lib/pulp``. The repository's relative url is
```

Comment:
The ``relative_url`` option added by this PR is documented in the ISO distributor documentation, yet this option is in the ISO rsync distributor documentation. Aren't the two distributors distinct, such that they don't reference each others options?